README.md

```diff
@@ -21,14 +21,22 @@ Configuring Squid
 
 The recommended way to configure the squid in the container is by means of environment variables.
 Three such variables are supported:
-* SQUID_IPRANGE - Limits the incoming connections to the provided whitelist. By default only standard private network addresses are whitelisted.
-* SQUID_CACHE_DISK - Sets the cache_dir option which determines the disk size squid uses. Must be an integer value, and its unit is MBs. Defaults to 10000.
-                     Note: The cache disk area is located at /var/cache/squid.
-* SQUID_CACHE_MEM - Sets the cache_mem option which regulates the size squid reserves for caching small objects in memory. Defaults to "128 MB".
+|---------------------|-------------------------------------------------------------------------|----------------------------------------------|
+| SQUID_IPRANGE       | Limits the incoming connections to the provided whitelist.     |By default only standard private network addresses are whitelisted. |
+| SQUID_CACHE_DISK    | Sets the cache_dir option which determines the disk size squid uses. Must be an integer value, and its unit is MBs. Note: The cache disk area is located at /var/cache/squid. | Defaults to 10000. |
+| SQUID_CACHE_MEM     | Sets the cache_mem option which regulates the size squid reserves for caching small objects in memory. | Defaults to "128 MB". |
+|---------------------|-------------------------------------------------------------------------|----------------------------------------------|
+
+Moreover, be aware that in order to preserve the cache between redeployments, you should map the following areas to persistent storage outside the container:
+|------------------|-------------------------------------------------------------------------------|
+| /var/cache/squid | This directory contains the cache for squid. See also SQUID_CACHE_DISK above. |
+| /var/log/squid   | This directory contains the squid logs.                                       |
+|------------------|-------------------------------------------------------------------------------|
```

Comment:
Could you add a sentence about how to actually do this via a volume mount example with docker run?

Reply:
No clue... But I can add a Kubernetes example.

Reply:
I'm not as interested in a k8s examples since the OSG doesn't officially support it. I think what the Slate team had in their previous table was good (`-v /tmp/squid:/var/cache/squid`) as long as we clarify that `/tmp/squid` is a dir on the host and this is an option to be added to `docker run`.

Reply:
Done